examples/freertos_plus_tcp/z_pub_st.c

```diff
@@ -54,7 +55,7 @@ void app_main(void) {
 
     char *buf = (char *)pvPortMalloc(256);
     z_clock_t now = z_clock_now();
-    for (int idx = 0; 1;) {
+    for (int idx = 0; idx < N;) {
```

Comment:
In this case after `10` publications the example will exit, however all other examples kepp publishing indefinitely.